test/catalog-fixtures/DefaultCatalogFixture.json

```diff
@@ -0,0 +1,229 @@
+{
+	"categories": [
+		{
+			"paths": [
+				"category/ecmascript/design.xml",
+				"category/apex/design.xml"
+			],
+			"name": "Design",
+			"engine": "pmd"
+		},
+		{
+			"paths": [
+				"category/ecmascript/bestpractices.xml",
+				"category/apex/bestpractices.xml"
+			],
+			"name": "Best Practices",
+			"engine": "pmd"
+		},
+		{
+			"name": "Best Practices",
+			"engine": "eslint",
+			"paths": [
+				"https://eslint.org/docs/rules/no-implicit-globals",
+				"https://eslint.org/docs/rules/no-implicit-coercion"
+			]
```

Comment:
I confess, it's slightly weird to me that eslint and pmd would have separate entries for a category with the same name, instead of just turning paths into an object whose keys are engine names and whose values are arrays of paths. I assume this way is to make it easier for RuleManager (or whichever) to dereference categories?

Reply:
I started that way, actually.  But we need to know that a particular path within a category applies to a particular engine.  So instead of changing the format to have an engine property  in each path value, I figured having separate categories for each engine, with the same category name, was simplest.